cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -113,6 +113,9 @@ function start-kube-apiserver {
   if [[ -n "${SERVICEACCOUNT_CERT_PATH:-}" ]]; then
     params+=" --service-account-key-file=${SERVICEACCOUNT_CERT_PATH}"
   fi
+  if [[ -f "/etc/srv/kubernetes/known_tokens.csv" ]]; then
```

Comment:
Better use single (or no) quotes for static strings. Suggestion:
```shell
if [[ -f '/etc/srv/kubernetes/known_tokens.csv' ]]; then
```

Reply:
Updated to use a variable as your suggested.